tests/integration/test_ignore.py

```diff
@@ -26,9 +26,9 @@ def test_ignore_localhost(tmpdir, httpbin):
     with overridden_dns({'httpbin.org': '127.0.0.1'}):
         cass_file = str(tmpdir.join('filter_qs.yaml'))
         with vcr.use_cassette(cass_file, ignore_localhost=True) as cass:
-            urlopen('http://localhost:{0}/'.format(httpbin.port))
+            urlopen('http://localhost:{}/'.format(httpbin.port))
```

Comment:
It think it's still a good practice to keep the index in the format. Just an observation, afaik pep8 or any other standard does not enforce anything for this.

Reply:
I can revert those if needed. I prefer to omit them if possible to aid readability and reduce cognitive load: if I see some numbers in there then I know it's a special case, like different ordering.

Reply:
It's just personal preference, and what you just said makes sense. In this case makes sense to keep the indexes out of it 😄